app/lib/common/widgets/attachments/attachment_item.dart

```diff
@@ -20,123 +23,18 @@ class AttachmentItem extends ConsumerWidget {
     if (type == 'image') {
       return AttachmentContainer(
         name: msgContent.body(),
-        child: _ImageAttachment(attachment: attachment),
+        child: ImageView(attachment: attachment, openView: openView),
```

Comment:
We can re-used full screen image view common dialog which is already used in chat media.
`import 'package:acter/common/widgets/image_dialog.dart';`

Reply:
The ```imageView``` uses that dialog. This widget is part of ```imageView```. See line 53.  
```
Widget imagePlaceholder(
    BuildContext context,
    Attachment attachment,
    AttachmentMediaState mediaState,
    WidgetRef ref,
  ) {
    final msgContent = attachment.msgContent();
    return InkWell(
      onTap: () async {
        if (mediaState.mediaFile != null) {
          showAdaptiveDialog(
            context: context,
            barrierDismissible: false,
            useRootNavigator: false,-
            builder: (ctx) => ImageDialog(                 <--- Component
              title: msgContent.body(),
              imageFile: mediaState.mediaFile!,
            ),
          );
        } else {
          ref
              .read(attachmentMediaStateProvider(attachment).notifier)
              .downloadMedia();
        }
      },
      ```